recipes/acpype/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "acpype" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+    git_rev: d99be8e17e3f999e06a401e5efdbe5f2fc9a2959
```

Comment:
Do not use git_url if not really needed. You can use: https://github.com/alanwilter/acpype/archive/d99be8e17e3f999e06a401e5efdbe5f2fc9a2959.tar.gz instead.

Reply:
Aha, thanks.

Reply:
Can you open an issue to ask the maintainers to tag the relevant commit when they make releases?

You can then use a predicable URL and the updater bot will be able to monitor for new releases.

Reply:
Done: https://github.com/alanwilter/acpype/issues/9

How should I write the URL? Could you link to an example?

Reply:
Normally something like this is good enough:
  - `https://github.com/alanwilter/acpype/archive/{{ version }}.tar.gz`
  - `https://github.com/alanwilter/acpype/archive/v{{ version }}.tar.gz`

It depends how the project decides to name the tag. There are fancier tricks that can be used for projects that use tags like `v0r1p0` or `v0_1_0`. If you need them ping me and I can try and help.

Reply:
Thanks; what about a tag based on a date like this? https://github.com/alanwilter/acpype/releases